src/main/java/io/vertx/gsoc2018/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -29,11 +31,12 @@ public void start(Future<Void> startFuture) throws Exception {
 
     Router router = Router.router(vertx);
 
-    // router.route().handler(BodyHandler.create());
+     router.route().handler(BodyHandler.create());
```

Comment:
Minor: you could restrict the body capture to `POST` requests

Reply:
It has been corrected in https://github.com/tsegismont/vertx-gsoc-2018-qotd/pull/4/commits/b52608764c1e761755d11a61264d24ef9911945b.